### DISCUSSION TITLE
How to set up Maple Font in VS Code in Ubuntu Budgie OS?

### DISCUSSION THREAD

**Discussion by piyush-aggarwal-cloudcraftz** [subframe7536/maple-font#292] [Q&A] [answered]
Hello Guys,

I am student and I need help in setting up Maple Mono font in VS Code. I am trying to replicate theme at:
_https://marketplace.visualstudio.com/items?itemName=subframe7536.theme-maple&ssr=false#review-details_

I have installed theme but fonts are not recognised by VS code. I have extracted and installed font in system but VS code is not picking them up. The user settings.json looks like:

```
{
    "workbench.colorTheme": "Maple Light",
    "files.autoSave": "onFocusChange",
    "editor.fontLigatures": true,
    "workbench.settings.applyToAllProfiles": [
        "editor.fontFamily"
    ],
    "json.schemas": [

    ],
    "editor.fontFamily": "'Maple Mono', 'Droid Sans Mono', 'monospace', monospace"
}
```

**Maple Font Version:** [V7.0 Beta30](https://github.com/subframe7536/maple-font/releases/tag/v7.0-beta30) Pre-release

Please help.
Thank You!!

**Comment by subframe7536** [comment]
If you make sure that you have installed `Maple Mono`, then try to setup `"editor.fontFamily": "Maple Mono, Droid Sans Mono, monospace"`

Also, if you are using MacOS, make sure your VSCode is totally closed (press `CMD + Q`) and then restart VSCode.

**Comment by piyush-aggarwal-cloudcraftz** [comment, marked as answer]
I found this solution:
I copied Maple Font folder to "/usr/share/fonts/opentype". After this VS Code was able to pick it up. I don't know why its not picking it up from "/home/user_name/.local/share/fonts/Maple Mono". Any idea how to fix this?
System: Ubuntu 24.04 LTS

**Reply by subframe7536** [reply]
I'm not familiar with linux desktop, maybe you can run `font-cache` to refresh your cache
